### ISSUE TITLE
Add Saving Throw buttons to the HUD itself [FEATURE]

### ISSUE THREAD

**Issue by orcnog** [Larkinabout/fvtt-token-action-hud-core#170]
**TL;DR:**
Split the Abilities buttons into two sub categories, "Ability Checks" and "Saving Throws", in order to eliminate an extra click in a far-off dialog. Mockup provided.

**Description**
Currently, I have to click on an ability button (ex: Dex), then hundreds of pixels away, a dialog pops up asking me to select Ability Check vs Saving Throw (this is out of the box 5e UI).  As rolling saving throws is *way* more common (at least for *me* in dnd5e) than rolling a raw ability check (which only occurs when rolling for something that *isn't* covered by proficiencies like Stealth, etc), I'd really appreciate not having to hunt for the correct button 1000px away and make that extra click each time I use the HUD to roll a save.  Why not just add another subcategory below "Abilities" labeled "Saving Throws" to eliminate the extra dialog?  Or if this is intended to be used in conjunction with some module that already eliminates that dialog, please list it in the Recommended Modules section of the README.
Personally, I use Ready Set Roll, which eliminates the additional advantage / normal /disadvantage dialog, but even for users who don't employ that module, eliminating an extra click is always appreciated.

**Versions**
- Token Action HUD System: token-action-hud-dnd5e
- System: dnd5e

**Additional context**

*Proposed*
![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/70491549/f50517ad-7a26-4d92-9b98-8b1f899e7d7e)

*Current*
![image](https://github.com/Larkinabout/fvtt-token-action-hud-core/assets/70491549/63bf1b40-b74e-40ab-8c7e-a6791e9e3970)

**Comment by Larkinabout** [comment]
Hi @orcnog, these already exist. The HUD is configurable and there are groups for Checks and Saving Throws. Check out [How to Use Token Action HUD](https://github.com/Larkinabout/fvtt-token-action-hud-core/wiki/How-to-Use-Token-Action-HUD)

**Comment by orcnog** [comment]
Amazing. Thanks for the knowledge drop.
